src/scratch/gangemesteren/gangemesteren_nn.md

```diff
@@ -106,39 +106,39 @@ La oss sjå om me kan setje saman desse tala til eit skikkeleg spørsmål.
 - [ ] Lag ein ny variabel, `spørsmål`{.blockdata}. Også denne skal gjelde for
       alle figurar.
 
-- [ ] Klossen `sett sammen`{.blockoperators} kan brukast for å setje saman
-      fleire tal og ord. Me skal nå bruka to `sett
-      sammen`{.blockoperators}-klossar på denne måten:
+- [ ] Klossen `set saman`{.blockoperators} kan brukast for å setje saman
+      fleire tal og ord. Me skal nå bruka to `set
+      saman`{.blockoperators}-klossar på denne måten:
 
     ```blocks
-    sett sammen (sett sammen [] []) []
+    set saman (set saman [] []) []
     ```
 
     Dette gir oss plass til tre tal eller ord. Her kan me putta inn
     `tal1`{.blockdata}, teksten ` gonger ` og `tal2`{.blockdata}. Pass på at du
     har mellomrom før og etter `gonger`, det ser best ut da. Om du klikkar på
-    den første `sett sammen`{.blockoperators}-klossen vil du sjå korleis den
+    den første `set saman`{.blockoperators}-klossen vil du sjå korleis den
     ferdige teksten blir.
 
-    ![Bilete av sett sammen-blokkane i Scratch](sett_sammen.png)
+    ![Bilete av set saman-blokkane i Scratch](sett_sammen.png)
```

Comment:
Det bilde her er litt dumt. For det er bilde av blokker på bokmål. Kanskje sette sammen sånn det står der i scratch. Tar printscrn og legge ved det i tillegg? Blir et bilde ekstra per språk da men...

Reply:
Eller skrive `... den ferdige teksten blir (merk at blokkene er på bokmål).`? Det er ein del arbeid viss me skal gå over alle bileta som har tekst på bokmål. @tjesi?

Reply:
eg er enig i at vi ikke trenger lage nye bilder hver gang det inneholder tekst på bokmål. derimot sjekket eg scratch-oppgavenenå, og dette var eneste tilfellet eg fant hvor det var bilde av selve kodeblokkene. eg foreslår at vi lager et nytt bilde denne gangen.